src/app/shared/tabs/entity-datatable-tab/datatable-multi-row/datatable-multi-row.component.scss

```
@@ -62,6 +62,10 @@
       overflow-wrap: anywhere;
     }
 
+    ::ng-deep .mat-mdc-header-cell.right .mat-sort-header-container {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)